CareKit/Connect/OCKConnectDetailViewController.m

```diff
@@ -153,21 +154,21 @@ - (void)makeCallToNumber:(NSString *)number {
     [[UIApplication sharedApplication] openURL:[NSURL URLWithString:stringURL]];
 }
 
-- (void)sendMessageToNumber:(NSString *)number {
+- (void)sendMessageToNumber:(NSString *)number withDelegate:(id<MFMessageComposeViewControllerDelegate>)messageDelegate usingPresentingViewController:(UIViewController *)presentingViewController {
     MFMessageComposeViewController *messageViewController = [MFMessageComposeViewController new];
     if ([MFMessageComposeViewController canSendText]) {
-        messageViewController.messageComposeDelegate = self;
+        messageViewController.messageComposeDelegate = messageDelegate;
         messageViewController.recipients = @[number];
-        [self presentViewController:messageViewController animated:YES completion:nil];
+        [presentingViewController presentViewController:messageViewController animated:YES completion:nil];
     }
 }
 
-- (void)sendEmailToAddress:(NSString *)address {
+- (void)sendEmailToAddress:(NSString *)address withDelegate:(id<MFMailComposeViewControllerDelegate>)emailDelegate usingPresentingViewController:(UIViewController *)presentingViewController {
```

Comment:
New naming convention deprecates use of prepositions. Change withDelegate to delegate, and usingPresentingViewController to presentingViewController.